devel/libmaxminddb/Portfile

```diff
@@ -39,7 +39,7 @@ build.env-append    CC=${configure.cc} \
                     "LDFLAGS=${configure.ldflags} [get_canonical_archflags ld]"
 
 depends_test-append \
-                    port:p5-ipc-run3
+                    port:p5.34-ipc-run3
```

Comment:
how did you make sure that is will use the `perl5.34` when running the tests and not the system version as Ryan pointed out in the Trac ticket?

Reply:
if it would be using the systems Perl version there would be no `ipc-run3` package installed so the tests would presumably fail. So I am pretty sure you will need to make sure the tests are using the correct Perl version - whether or not that is the case automagically you'll have to figure out.

Reply:
I decided to revert the change I made in 2023 to add `IPC::Run3`. That specific test is skipped over the missing module and it has no impact on the final result. The amount of work required seems significant compared to the minor benefit gained by that exercise.